src/main/java/homework/seven/ArrayDataException.java

```diff
@@ -0,0 +1,19 @@
+package homework.seven;
+
+public class ArrayDataException extends Exception {
```

Comment:
Typically you extends  RuntimeExceptions

Reply:
i fixed on RuntimeExceptions ArrayDataException extends RuntimeException